main.js

```diff
@@ -1,60 +1,58 @@
-const quotes = [
```

Comment:
nice work, @MatthewBerglund. Nice and readable. No comments from my side. Do you want to merge or shall I?

Reply:
Thanks, @motine. I think I'm going to first revert back to a version of the code that fetches `quotes.json` instead of the external API because of the inappropriate language issue I mentioned before. Then, I'll rewrite the code to use async/await instead of `fetch().then().then()` . 

Question: Should I merge first and then replace the external API? Or replace and then merge?

Reply:
whohoo, I have an idea. what if we have multiple branches (good exercise with git)... 
the master branch will be the version you want to show off. then we have one branch with the fetch-version and one with the async-version.

approach:
1. merge this merge request without fetch changes to master
2. create a new branch, directly commit your changes for fetch into this branch
3. create a new branch from the fetch branch (A), create yet another branch (B), apply the async changes to B. make a pull request from B to A.

these approaches should exercise your git-muscles quite a bit because we are doing 3 different approaches here.

pro tipp copy your repo into another folder on your harddrive if you screw something up.

Reply:
Okay, sounds good!